docs/library/pyb.Timer.rst

```diff
@@ -32,6 +39,10 @@ Further examples::
 the servo driver, and Timer 6 is used for timed ADC/DAC reading/writing.
 It is recommended to use the other timers in your programs.
 
+*Note:* Since even the exception handler can't allocate memory during interrupts,
+in order to get good traceback inside the interrupt timers use:
+``import micropython; micropython.alloc_emergency_exception_buf(100)``
```

Comment:
Again, as this change is targeted at novice users, would make sense to be more detailed where this code should be put. Even I would get an idea that it should be put either inside timer callback, or exception handler altogether.